extensions/2.0/Khronos/KHR_texture_basisu/README.md

```diff
@@ -0,0 +1,154 @@
+# KHR_texture_basisu
+
+## Contributors
+
+* Members of 3D Formats Working Group
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds the ability to specify textures using KTX2 images with Basis Universal supercompression. An implementation of this extension can use such images as an alternative to the PNG or JPEG images available in glTF 2.0 for more efficient asset transmission and reducing GPU memory footprint. Furthermore, specifying mip map levels is possible.
+
+When the extension is used, it's allowed to use value `image/ktx2` for the `image.mimeType` property.
```

Comment:
I'm not sure how to word this properly, but I don't think we should simply allow image/ktx2 for images. It should only be allowed when pointed to from a `KHR_texture_basisu` extension.

Reply:
Just confirming your meaning — `image.mimeType = "image/ktx2"` would be allowed ONLY IF that image is referenced by a `texture.extensions.KHR_texture_basisu.source`; the image cannot be referenced by `texture.source`? Or are you saying we need something like `image.extensions.KHR_texture_basisu.mimeType = "image/ktx2"` here?

Reply:
The former.

Reply:
Please review the updated language.

Reply:
LGTM